test/error.js

```diff
@@ -54,17 +63,27 @@ test('dns message', async t => {
 
 test('options.body error message', async t => {
 	const err = await t.throws(got(s.url, {body: {}}));
-	t.regex(err.message, /The `body` option must be a stream\.Readable, string or Buffer/);
+	t.is(err.message, 'The `body` option must be a stream.Readable, string or Buffer');
```

Comment:
We should change all of the `t.throws` assertions to the new format. Should not be done in this PR though. Just a note to myself.

Reply:
Ok, so after this gets merged I'll do another PR for that.